servers/text_server.h

```diff
@@ -390,7 +390,7 @@ class TextServer : public RefCounted {
 	virtual void font_set_opentype_feature_overrides(const RID &p_font_rid, const Dictionary &p_overrides) = 0;
 	virtual Dictionary font_get_opentype_feature_overrides(const RID &p_font_rid) const = 0;
 
-	virtual Dictionary font_supported_feature_list(const RID &p_font_rid) const = 0;
+	virtual Dictionary font_supported_feature_list_compiled(const RID &p_font_rid) const = 0;
```

Comment:
I guess it's a search/replace result, there's nothing to preprocess at compile tine here, it's a features list of the specific font that can be determined only after the font is loaded.

Reply:
I didnt changed this file, what your change has to do with the PR, sorry didnt got it!